Px.Utils/Validation/ValidationObject.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace PxUtils.Validation
+{
+    /// <summary>
+    /// Defines an abstract class for validation objects. A validation object represents a piece of data that is subject to validation.
+    /// </summary>
+    public abstract class ValidationObject(int line, int character, string file)
+    {
+        /// <summary>
+        /// Line number in the file where the entry is located.
+        /// </summary>
+        public int Line { get; } = line;
```

Comment:
Entry can be on multiple lines, how is that handled?

Reply:
It is the line where the entry starts

Reply:
Line number is incremented when entries are build at every line break and entries are built when a delimiter is found. Thus the line number associated with each entry is the last line that is associated with that entry.